src/components/implementation/tests/micro_llping/ping.c

```diff
@@ -6,10 +6,16 @@
 #include <stdio.h>
```

Comment:
Not sure this should really be called ping here.  Isn't it the `parmem_mgr` test?